opal/browser/dom/node.rb

```diff
@@ -190,11 +190,6 @@ def remove
     parent.remove_child(self) if parent
   end
 
-  # Remove all the children of the node.
-  def clear
```

Comment:
I removed it because I wasn't sure if it will work as `children` is read-only, but seems individual elements can still be removed. Also previously it would remove all child nodes, but now this would remove only element nodes. What is expected behavior?